tests/baselines/reference/jsDeclarationsParameterTagReusesInputNodeInEmit1.errors.txt

```diff
@@ -14,17 +16,29 @@ tests/cases/conformance/jsdoc/declarations/file.js(8,1): error TS9006: Declarati
     
     module.exports = BaseFactory;
     
-==== tests/cases/conformance/jsdoc/declarations/file.js (1 errors) ====
+==== tests/cases/conformance/jsdoc/declarations/file.js (3 errors) ====
     /** @typedef {import('./base')} BaseFactory */
+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS4081: Exported type alias 'BaseFactory' has or is using private name 'Base'.
+    /**
+     * @callback BaseFactoryFactory
+       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+     * @param {import('./base')} factory
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+     */
+    ~
+!!! error TS4081: Exported type alias 'BaseFactoryFactory' has or is using private name 'Base'.
+    /** @enum {import('./base')} BaseFactoryFactoryRegistry */
+        ~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS4081: Exported type alias 'Base' has or is using private name '"tests/cases/conformance/jsdoc/declarations/base"'.

```

Comment:
This one seems less like the other two.

Reply:
uhhhhhhhh yeah, I didn't notice that. It should be `${0}=BaseFactoryFactoryRegistry, ${1}='Base'`.